.github/actions/install_requirements/action.yml

```
@@ -15,7 +15,8 @@ runs:
       run: |
         PYTHON_VERSION="${{ inputs.python-version }}"
         if [ $PYTHON_VERSION == "dev" ]; then
-          PYTHON_VERSION=$(sed -n "s/ARG PYTHON_VERSION=//p" Dockerfile)
+          # python version from Dockerfile, removing potential pinned sha
```

` removing potential pinned sha` 

probably you mean `pinned sha hash` as 'sha' means just secure hashing algorithm and we are not pinning the algorithm - I'd 
```suggestion
          # python version from Dockerfile, removing potential pinned commit hash
```

---

apparently I'm wrong and `sha` is used in this context - however [there ](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/about-commits#about-commits) as well as in one other place in the codebase we use uppercase SHA

---

still 'hash' might be clearer